CHANGELOG.md

```diff
@@ -67,6 +67,8 @@
 
   Note that this feature is mainly for local scripting. Most deploy providers don't let you upload `.env` files (unless you're using baremetal) and usually have their own way of determining environments.
 
+- feature: Enable CSS nesting syntax by default when using Tailwind
```

Comment:
This changelog entry doesn't really highlight this feature, and it's a cool/handy one. Your PR body was nice since it included a code sample. Not sure if you saw the failing action, but it should've included your original PR body as a starting point. Mind fleshing this out?